From a00f6f163a3f4fd6fe8ff2d821fae286e3a872cf Mon Sep 17 00:00:00 2001
From: Felix Riemann <friemann@gnome.org>
Date: Mon, 24 Aug 2015 19:37:16 +0200
Subject: map: Use a non-deprecated image icon

"gnome-mime-image" requires the deprecated gnome-icon-theme
package. Use "image-x-generic" instead, which is part of adwaita-icon-theme.

https://bugzilla.gnome.org/show_bug.cgi?id=753761

diff --git a/plugins/map/eog-map-plugin.c b/plugins/map/eog-map-plugin.c
index 4b68bbd..9f8227d 100644
--- plugins/map/eog-map-plugin.c
+++ plugins/map/eog-map-plugin.c
@@ -67,7 +67,7 @@ update_marker_image (ChamplainLabel *marker,
 	thumb = gtk_clutter_texture_new ();
 	gtk_clutter_texture_set_from_icon_name (GTK_CLUTTER_TEXTURE (thumb),
 						widget,
-						"gnome-mime-image",
+						"image-x-generic",
 						size, NULL);
 	/* don't need to unref widget because it is floating */
 
-- 
cgit v0.10.2

